src/client.rs

```diff
@@ -72,7 +72,10 @@ mod encryption {
             Mode::Tls => {
                 let config = {
                     let mut config = ClientConfig::new();
-                    config.root_store.add_server_trust_anchors(&webpki_roots::TLS_SERVER_ROOTS);
+                    config.root_store = match rustls_native_certs::load_native_certs() {
+                        Ok(store) => store,
+                        Err((_, err)) => Err(err)?,
+                    };
```

Comment:
IMHO it would look a bit prettier if we could write it like this:
```suggestion
                    config.root_store = rustls_native_certs::load_native_certs().map_err(|(_, err)| err)?;
```

Reply:
I agree that does look better. Though something I forgot to mention is that `load_native_certs()` returns `Result<RootCertStore, (Option<RootCertStore>, Error)>`, the `Option<RootCertStore>` can include certs that are successfully loaded. Do we want to fallback to that in case of an error and log the error, or do we want to keep the behaviour as-is (don't fallback to partially loaded certs, and return the error)?

Reply:
I think failing in case of an error would be fine. I expect that the `TlsError` returned to the user should be sufficient to understand what caused an error. Perhaps we could think of an improvement while working on https://github.com/snapview/tungstenite-rs/issues/203 which looks like a nice thing to have.